source/sass/buyers-guide/views/homepage.scss

```diff
@@ -135,9 +135,15 @@
     @media (min-width: $bp-lg) {
       @include product-box-size(1px, 4);
     }
+  }
 
+  .project-list-section {
     @media (min-width: $bp-xl) {
-      @include product-box-size(1px, 5);
+      // on extra large screen we want project-list-section's width to match
+      // Bootstrap's .container's width
+      width: calc(#{map-get($container-max-widths, xl)} - #{map-get($grid-gutter-widths, xl)});
+      max-width: 100%;
+      margin: 0 auto;
```

Comment:
If there are margin requirements we should probably use an `m-...` rule in the HTML

Reply:
Yea I thought about it.  But since we are adding styling rules only for XL screens, I think leaving them all in homepage.scss makes it easier to glance vs adding `mx-xl-auto` class in DOM that deals with horizontal margin + the rest of the rules in .scss